data/store/mongo/mongo_devicetokens.go

```diff
@@ -0,0 +1,59 @@
+package mongo
```

Comment:
This should be moved to `auth`

Reply:
> This should be moved to `auth`

This already was moved, I just forgot the `git rm` this. Thanks, and done.